src/test/java/uk/gov/hmcts/reform/professionalapi/controllers/request/OrganisationCreationRequestValidatorTest.java

```diff
@@ -34,5 +35,9 @@ public void calls_all_validators() {
 
         verify(validator1, times(1)).validate(request);
         verify(validator2, times(1)).validate(request);
+
+        assertThat(OrganisationCreationRequestValidator.contains("pending")).isEqualTo(true);
```

Comment:
See above comment. There is no need to store values in the enum. And when using enums there is no need to use strings.